pkg/cmd/roachtest/roachtestutil/mixedversion/planner_test.go

```diff
@@ -556,7 +558,7 @@ func Test_stepSelectorFilter(t *testing.T) {
 			name:                   "no filter",
 			predicate:              func(*singleStep) bool { return true },
 			expectedAllSteps:       true,
-			expectedRandomStepType: runHookStep{},
```

Comment:
This was basically undoing the change in [1] since we reverted the logic behind finding upgrade paths.

[1] https://github.com/cockroachdb/cockroach/pull/142211